jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -94,8 +95,11 @@ public static Containerizer to(DockerDaemonImage dockerDaemonImage) {
         ImageConfiguration.builder(dockerDaemonImage.getImageReference()).build();
 
     DockerClient dockerClient =
```

Comment:
I think `to(DockerDaemonImage)` should delegate to `to(DockerClient, DockerDaemonImage)` - WDYT @eddumelendez?